nextcloud-official.json

```
@@ -0,0 +1,40 @@
+{
+    "Nextcloud-Official": {
+        "containers": {
+            "nextcloud-official": {
+                "image": "nextcloud",
+                "launch_order": 1,
+                "ports": {
+                    "80": {
+                        "description": "Nextcloud WebUI port. Suggested default: 8080",
+                        "host_default": 8888,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/var/www/html": {
+                        "description": "Choose a Share for Nextcloud. Eg: create a Share called nextcloud-all for this purpose alone.",
+                        "label": "App Storage",
+                        "min_size": 1073741824
+                    }
+                },
+                "environment": {
+					"SQLITE_DATABASE": {
+						"description": "Enter the name of the database you would like to create.",
+						"label": "SQLite DB Name",
+						"index": 1
+					}
+				}
+            }
+        },
+	    "version": "latest",
+        "description": "Next generation open source Enterprise File Sync and Share",
+        "icon": "https://upload.wikimedia.org/wikipedia/commons/6/60/Nextcloud_Logo.svg",
+        "more_info": "<p>No smbclient installed by default.  If you would like to enable this you can connect to your container and use <code>apt-get install smbclient -y</code> to install.</p>",
+        "ui": {
+            "slug": ""
+        },
```

Add support for "Add Storage" post-install:
```suggestion
        },
        "volume_add_support": true,
```